src/components/Header/Header.tsx

```diff
@@ -85,10 +83,17 @@ export const Header = ({
   const AsideTypography = platform === Platform.VKCOM ? Subhead : Text;
   const SubtitleTypography = mode === "secondary" ? Subhead : Caption;
 
+  const isTappable = !!restProps.onClick || !!restProps.href;
```

Comment:
```suggestion
  const isTappable = Boolean(restProps.onClick || restProps.href)
```
Может, так более читаемо?

Reply:
✅